src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -201,10 +329,9 @@ private IEnumerable<Mutant> ApplyMutator(SyntaxNode syntaxNode, IMutator mutator
             }
         }
 
-        private T ApplyMutant<T>(T node, Mutant mutant) where T: SyntaxNode
+        private static T ApplyMutant<T>(T node, IReadOnlyMutant mutant) where T: SyntaxNode
```

Comment:
Why did you make this static?

Reply:
It is static as it is nothing more than syntactic sugar. Marking it as static makes it explicit: no side effects.

Reply:
Actually, it looks like it should be Mutant method.